correios/utils.py

```diff
@@ -56,7 +56,7 @@ def __init__(self, *ranges):
 
             try:
                 element = element or [range(*r)]
-            except:
+            except TypeError:
```

Comment:
Put this in a separated PR. I think @gilmarsoares-luizalabs will fix this issue too.